Pod/Classes/PINRemoteImageManager.m

```diff
@@ -132,6 +132,15 @@ @implementation PINRemoteImageManager
 
 static PINRemoteImageManager *sharedImageManager = nil;
 static dispatch_once_t sharedDispatchToken;
+static BOOL supportsQOS = NO;
+
++ (void)load
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
```

Comment:
Maybe better to do this in a method like +supportsQOS, and then just trigger it on first access rather than +load?